### PR TITLE
Update addon-resizer README regerding estimator flag.

### PR DESCRIPTION
**Pull request by krzysied**
Update addon-resizer README

### CODE REVIEW

**Review by jbartosik — COMMENTED** [review]
One suggestion mostly looks good to me

**Review by krzysied — COMMENTED** [review]
.

File: addon-resizer/README.md [hunk 1]
```diff
@@ -30,6 +31,10 @@ Usage of pod_nanny:
       --use-metrics=false: Whether to use apiserver metrics to detect cluster size instead of the default method of listing node objects from the Kubernetes API.
 ```
 
+Estimators:
+- linear - based on the current number of nodes.
+- exponential - current number of nodes if substituded with the first greater number from the geometric sequence (a=minClusterSize, r=1.5).
```
[the remark this answers is not in the document]
Comment (krzysied) [reply addon-resizer/README.md]: Done. I've mentioned that it is "greater than or equal to".

**Review by jbartosik — APPROVED** [review]
/lgtm
/approve

**Pull request merged**